TESTING.md

```diff
@@ -90,6 +139,46 @@ This entity is initialized when the test starts. You can assume that the variabl
 The `check` variable is our interface with the `check.v1` package. We can do several things with it, such as skipping a test, probing a condition, declare success or failure, and more.
 
 
+### Adding build tags.
+
+All tests need to have a build tag. The tag should be the first line of the file, followed by a blank line
+
+```go
+// +build functional featurename ALL
+
+package govcd
+```
+
+Tests that integrate in the functional suite use the tag `functional`. Using that tag, we can run all functional tests
+at once.
+We define as `functional` the tests that need a live vCD to run.
+
+Note that as of today (April 2019) we only have functional tests, but we plan to add more, which can be, for example,
```

Comment:
Please remove "(April 2019)" - it will get old quickly. "as of today" is enough ;)

Reply:
fixed